src/main/java/com/dreammaster/bartworksHandler/BioItemLoader.java

```diff
@@ -116,7 +116,7 @@
                 new FluidStack[]{new FluidStack(BIOFLUIDS[3],50)},
                 1200,BW_Util.getMachineVoltageFromTier(8),100,8,0,false
         );
-        for (int i = 0; i < OreDictionary.getOres("cropTcetiESeaweed").size(); i++) {
+        for (int i = 1; i < OreDictionary.getOres("cropTcetiESeaweed").size(); i++) {
```

Comment:
Shouldn't this be .size() + 1? Otherwise you miss out on looping with the final ore